.github/workflows/release.yml

```diff
@@ -22,8 +23,17 @@ jobs:
 
       - name: Install Dependencies
         run: npm i
+        
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
```

Comment:
I believe `set-output` is deprecated -> https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

You should just use `GITHUB_OUTPUT`

```
- name: Set output
  run: echo "{name}={value}" >> $GITHUB_OUTPUT
```

Reply:
I checked the docs, it looks like it's available in `${{ github.ref_name }}` now without me needing to parse it.